css/style.css

```diff
@@ -9,400 +14,622 @@ body {
-          height: 37px;
-          padding: 5% 0 0 7%; 
-        }
-    }  
-
-@media (max-width: 799px) { 
-    .girl {
-        background: none
-     }
+  .logo-mobil {
+    position: absolute;
+    width: 98px;
+    height: 37px;
+    padding: 5% 0 0 7%;
     }
-
+}  
+ 
 @media (max-width: 799px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: 0 92px; 
-        width: 80%;
-        margin-left: 10%;
-        position: absolute;
-        
-    }
- }
-
- @media (max-width: 799px) { 
-    .title {
-        top: 80%;
-        width: 15%;
-        font-size: 50px;
-        line-height: 52px;
-        left: 30%;
+  .girl {
+    background: none;
     }
-     } 
-
+}
+ 
+@media (max-width: 799px) { 
+  .new-сollection {
+    background: url(../img/girl2.jpg) no-repeat;
+    background-size: 125%;
+    background-position: 0 92px;
+     }
+}
+ 
+@media (max-width: 799px) {
+  .title {
+    top: 80%;
+    width: 15%;
+    font-size: 50px;
+    line-height: 52px;
+    left: 30%;
+     }
+} 
+ 
 @media (max-width: 800px) {      
-     .galka  {
-        position: absolute;
-        left: 45%;
-        top: 95%;
-    }
+  .galka  {
+    position: absolute;
+    left: 45%;
+    top: 95%;
+     }
 } 
-
+ 
 @media (max-width: 630px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: -50px 92px; 
-    }
- }
-
- @media (max-width: 630px) { 
-    .title {
-        top: 83%;
-        width: 15%;
-        font-size: 40px;
-        line-height: 42px;
-        left: 20%;
-    }
-     } 
-
+  .new-сollection {
+    background: url(../img/girl2.jpg) no-repeat;
+    background-size: cover;
+    background-position: -50px 92px; 
+     }
+}
+ 
+@media (max-width: 630px) { 
+  .title {
+    top: 83%;
+    width: 15%;
+    font-size: 40px;
+    line-height: 42px;
+    left: 25%;
+     }
+} 
+ 
 @media (max-width: 630px) {      
-     .galka  {
-        position: absolute;
-        left: 45%;
-        top: 95%;
-    }
+  .galka  {
+    position: absolute;
+    left: 45%;
+    top: 95%;
+     }
 } 
-
-@media (max-width: 499px) { 
-    .title {
-        top: 65%;
-        font-size: 25px;
-        line-height: 22px;
-    }
-     } 
-
-@media (max-width: 499px) {      
-     .galka  {
-        left: 40%;
-        top: 75%;
-    }
+ 
+@media (max-width: 440px) { 
+  .title {
+    top: 65%;
+    font-size: 25px;
+    line-height: 22px;
+    left: 30%;
+     }
 } 
-
-@media (max-width: 499px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-position: 20% 36%; 
-        margin-left: 10%;
-    }
- }
+ 
+@media (max-width: 440px) {      
+  .galka  {
+    left: 40%;
+    top: 75%;
+     }
+} 
+ 
+@media (max-width: 440px) { 
+  .new-сollection {
+    background: url(../img/girl2.jpg) no-repeat;
+    background-position: -25px 36%; 
+     }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)